src/components/CreateResetPassword/CreateResetPasswordControl.js

```diff
@@ -1,15 +1,15 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
 import { connect as reduxConnect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
```

Comment:
Why do we need this import?

Reply:
After we changed the routing logic, we don't need it, missed that, thank you